modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/client/extension/ClientExtension.java

```diff
@@ -46,13 +46,25 @@ public Map<String, Object> toJSONMap(String pid) {
 
 		Map<String, Object> configMap = new HashMap<>();
 
-		configMap.put("baseURL", "${portalURL}/o/" + projectName);
+		configMap.put(
+			"baseURL",
+			typeSettings.getOrDefault(
+				"baseURL", "${portalURL}/o/" + projectName));
 		configMap.put("description", description);
-		configMap.put("dxp.lxc.liferay.com.virtualInstanceId", "default");
+		configMap.put(
```

Comment:
Hey Ray, I proposed handling this property in a different way here: https://github.com/brianchandotcom/liferay-portal/pull/133078
specifically this way:
https://github.com/brianchandotcom/liferay-portal/pull/133078/files#diff-5e3deeb76842bbeeb314a4735f3a0fecf5e96811d81e7cfa3563ffcb38ad08e2R97

I think we could remove this code where it grabs it from typesettings and instead just let jackson mapping handle it.

Reply:
Devil's advocate here... do we keep mapping properties (you can see I added several more)?

I'm a little remiss to have 2 different ways of handling all these properties. 

I'll change to that pattern and ideally it'll merge cleanly.